march_hardware/test/TestEncoder.cpp

```diff
@@ -96,6 +96,16 @@ TEST_F(TestEncoder, UpperSoftLimitHigherThanUpperHardLimit)
                march::error::HardwareException);
 }
 
+TEST_F(TestEncoder, ZeroPositionRadToZeroPosition)
+{
+  ASSERT_EQ(this->encoder.fromRad(0.0), this->zero_position);
+}
+
+TEST_F(TestEncoder, ZeroPositionToZeroRadians)
+{
+  ASSERT_DOUBLE_EQ(this->encoder.toRad(this->zero_position), 0.0);
```

Comment:
Also a nonzero test maybe :innocent: this will probably pass if you return nothing